functions.inc.php.php

```diff
@@ -11,7 +11,7 @@
 session_regenerate_id();
 if(!isset($_SESSION['csrf'])) csrf_gen();
 
-if(file_exists("db.inc") && file_exists("config.php")) unlink("config.php");
+if(file_exists("db.inc.php") && file_exists("config.php")) unlink("config.php");
 
 // Display basic website head section + body beginning
 function showHead($subname, $header = "Ambasadorzy Jawności") {
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [showHead accesses the super-global variable $_SERVER.](https://www.codacy.com/app/naczelnik/ambasadorzy-siecobywatelska-pl/file/1479063168/issues/source?bid=2991913&fileBranchId=2999032#l17)